.github/workflows/client-pr.yml

```diff
@@ -9,6 +9,12 @@ on:
     types: [opened, synchronize]
 
 jobs:
+  # SNYK auto PRs does not always update yarn.lock file
+  check-lock-file:
+    name: Update yarn.lock for Snyk updates
+    if: startsWith(github.head_ref, 'snyk-')
+    uses: ./.github/workflows/update-lockfile.yml
```

Comment:
you cant call a workflow. we need to write it as a custom action.

move file to a folder called update lock file.
add a file called action.yml